src/accel/detail/IntegrationSingleton.hh

```diff
@@ -6,21 +6,30 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include <memory>
 #include "accel/LocalTransporter.hh"
 #include "accel/SetupOptions.hh"
-#include "accel/SetupOptionsMessenger.hh"
 #include "accel/SharedParams.hh"
 
 namespace celeritas
 {
+//---------------------------------------------------------------------------//
+class ScopedMpiInit;
+class SetupOptionsMessenger;
+
 namespace detail
 {
 //---------------------------------------------------------------------------//
 /*!
  * Singletons used by the Integration interfaces.
  *
- * The single singleton instance contains global data. Thread-local data is
- * managed by the \c local_transporter static class.
+ * The singleton instance contains global data, including a single copy of the
+ * options (which are referenced by the UI setup) and . Thread-local data is
```

Comment:
Is there  a stray period in `and . Thread-local`?

Reply:
I think it's an infamous "squirrel comment": I got distracted while writing